content/hi/docs/home/supported-doc-versions.md

```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध प्रलेखन संस्करण 
```

Comment:
```suggestion
title: प्रलेखन के उपलब्ध संस्करण
```

This makes sentence more easy to read and understand.

Reply:
@verma-kunal
Can we commit these suggestions if you agree with the changes. We will be ready to merge this PR.

Reply:
I am not able to commit these!! its showing `Outdated suggestion`!! 
should i do this manually?